### ISSUE TITLE
Enhancement|Issue: Zooming in the animation window.

### ISSUE THREAD

**Issue by christianclavet** [BearishSun/BansheeEngine#322]
There is a working feature of the animation windows that work great: When we open the animation window, it show the full animation range (time and value) and is a desired behavior. 

There is the mouse wheel zoom feature, that could be desired in some case, but most of the time, we need to scale the time and value independently.

Here is a some suggestions that could improve working with this windows and make it easier to work with:

1- Add a auto-zoom range button for the current property displayed.
2- Ability to box select a range in the animation drawing to zoom in better
3- Add modifiers keys with the mouse wheel to scale the time or the value range.
4- Have data properties GUI like the inspector for the keyframes/events (displayed all the time and can change the values), and be able to go to the next keyframe using keys (like arrow keys or|and buttons on the interface (like the next frame, previous frame buttons)). Could use these to also jump from an event to the next. 

Not having at least the auto-zoom range button cause some issue, as I have to close the windows and open it again to see the range of my animation.

**Comment by BearishSun** [comment]
I agree using the zoom is inconvenient. For the auto-zoom, you can re-select the property in the property list (left part of the animation window) and it should auto-zoom, but I agree there should be a separate button for this.

**Comment by christianclavet** [comment]
Humm. I've tried it and it doesn't seem to auto-zoom. It only does it when I close and open back the animation window.
